introduction.md

```diff
@@ -1,3 +1,15 @@
 Hello! my name is Tania and this is my first time coming to this dojo. Namaste
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?